convert2rhel/utils.py

```diff
@@ -501,21 +501,35 @@ def download_pkg(
     if ret_code != 0:
         loggerinst.warning("Output from the yumdownloader call:\n%s" % (output))
 
-        if not "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" in os.environ:
-            loggerinst.critical(
+        # add in the new envar that should be changed to along with the old one
+        allow_old_envar_names = ("CONVERT2RHEL_INCOMPLETE_ROLLBACK", "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK")
+
+        # search for either envar in the system to see if it has been set
+        if any(envvar in os.environ for envvar in allow_old_envar_names):
+
+            # check to see which envar was used
+            if "CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK" not in os.environ:
+                # give the warning about the old envar not being used anymore if it is uesed
+                loggerinst.warning(
+                    "You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK'"
+                    " environment variable.  Please switch to 'CONVERT2RHEL_INCOMPLETE_ROLLBACK' instead."
+                )
+
+                # give regular warning if the new envar is used
+            loggerinst.warning(
```

Comment:
```suggestion
            # give regular warning if the new envar is used
            loggerinst.warning(
```

Reply:
didn't even realize i left that whitespace, I will get that taken care of as well, Thank you!